scenarios/cicd/start.sh

```diff
@@ -0,0 +1 @@
+#!/bin/bash
```

Comment:
I think this can b removed if not needed.

Reply:
Yeah it's just a good practice I generally follow to ensure my scripts get interpreted by bash (and not another shell like sh)

Reply:
Oh I guess I meant the start.sh file I don't think needs to exist if it's empty, but I could be wrong about that.